vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/treegrid/it/TreeGridHugeTreeNavigationIT.java

```diff
@@ -68,9 +68,6 @@ public void keyboard_navigation() {
 
         // Should collapse "Dad 1/1"
         new Actions(getDriver()).sendKeys(Keys.SPACE).perform();
-        // Make sure the "Dad 1/1" still has focus after collapsing (iron-list
-        // regression)
-        getTreeGrid().getCell(3, 0).focus();
```

Comment:
The workaround is no longer needed after [these fixes](https://github.com/vaadin/web-components/pull/389/files) in the Web Component